### PR TITLE
Enrich payload of subscribe $SYS event

### PR DESCRIPTION
**Pull request by behrad**

### CODE REVIEW

**Comment by mcollina** [comment]
can you please add a unit test?
Il giorno sab 18 lug 2015 alle 21:25 Behrad <notifications@github.com> ha
scritto:

**Comment by behrad** [comment]
adding tests, will push in minutes...

File: lib/server.js [hunk 1]
```diff
@@ -253,7 +253,20 @@ function Server(opts, callback) {
   that.on("subscribed", function(topic, client) {
     that.publish({
       topic: "$SYS/" + that.id + "/new/subscribes",
-      payload: client.id
+      payload: JSON.stringify({
```
Comment (behrad) [inline lib/server.js]: these are not in master! seems they are blown. This makes https://github.com/mcollina/mosca/pull/315 not to pass

Comment (mcollina) [inline lib/server.js]: what? something wrong happened during the merge. Can you add them back in #315?

**Pull request merged**